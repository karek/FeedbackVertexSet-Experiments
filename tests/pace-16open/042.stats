67,95,7
