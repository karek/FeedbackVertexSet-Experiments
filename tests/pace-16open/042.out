11
25 49 65 20 13 0 5 19 34 45 62 
